lotify/client.py

```diff
@@ -170,3 +171,42 @@ def __check_error(response):
             pass
         else:
             raise ValueError(response.json())
+
+def main(args):
+    client = Client()
+    if args.t:
+        access_token=args.t
+    if args.m and (not args.imgF):
+        message=args.m
+        response=client.send_message(access_token,message)
+        print(response)
```

Comment:
If you have implemented it, please remove it.

Reply:
Please tell me your reason ,thank you

Reply:
oh, I just mean `print()`, sorry

Reply:
I have completed .